plugins/mqtt/src/util.ts

```diff
@@ -0,0 +1,227 @@
+import convert from 'color-convert'
```

Comment:
I think color-util.ts would be a better filename

Reply:
done